V1/auth/validate.py

```diff
@@ -0,0 +1,58 @@
+from flask import make_response, jsonify
+import re
+
+
+def response(status, message, status_code):
+    """
+    Helper method to make an Http response
+    :param status: Status
+    :param message: Message
+    :param status_code: Http status code
+    :return:
+    """
+    return make_response(jsonify({
+        'status': status,
+        'message': message
+    })), status_code
+
+
+def empty(**data):
+    '''method to validate username input'''
```

Comment:
A good example of a comment within a function would be """`Takes in a `username` and returns a `boolean ` `"""